build/integration-test.sh

```diff
@@ -62,6 +65,10 @@ case "${1}" in
         # Run only part of apps
         TEST_APPS=${SHORT_APPS}
         ;;
+    oc)
```

Comment:
[minor] oc stands for openshift CLI. Better to rename this to `openshift` or `openshift-apps`